<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IPTVUtils</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="recordTab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Record</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="RecordWidget" name="recordWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="playbackTab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Playback</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="PlaybackWidget" name="playbackWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="convertTab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Convert</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="ConvertWidget" name="convertWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlaybackWidget</class>
   <extends>QWidget</extends>
   <header>playbackwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecordWidget</class>
   <extends>QWidget</extends>
   <header>recordwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConvertWidget</class>
   <extends>QWidget</extends>
   <header>convertwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
